evaluator/evaluate.go

```diff
@@ -52,10 +53,9 @@ func configure() {
 	enableRepoAnalysis = utils.GetBoolEnvOrDefault("ENABLE_REPO_ANALYSIS", true)
 	enableStaleSysEval = utils.GetBoolEnvOrDefault("ENABLE_STALE_SYSTEM_EVALUATION", true)
 	enableLazyPackageSave = utils.GetBoolEnvOrDefault("ENABLE_LAZY_PACKAGE_SAVE", true)
-	if enableLazyPackageSave {
-		ConfigurePackageNameCache()
-	}
 	enableBypass = utils.GetBoolEnvOrDefault("ENABLE_BYPASS", false)
+	memoryPackageCache = NewPackageCache()
+	memoryPackageCache.load()
```

Comment:
Shouldn't we think of reloading the cache sometimes to keep it up to date?

Reply:
It's updated by adding new items in lazySavePackages()/updatePackageCache()